src/pages/payment-methods/index.js

```diff
@@ -92,81 +92,100 @@ const PaymentMethods = () => (
-                            <AccordionItem
-                                key={idx}
-                                style={{
-                                    padding: '2.2rem 4.8rem',
-                                    position: 'relative',
-                                    paddingBottom: pd.note ? '5rem' : '2.2rem',
-                                }}
-                                parent_style={{
-                                    marginBottom: '4rem',
-                                }}
-                                header={pd.name}
-                            >
-                                <StyledTable has_note={!!pd.note}>
-                                    <Thead>
-                                        <Tr>
-                                            <Th>
-                                                <BoldText>{localize('Method')}</BoldText>
-                                            </Th>
-                                            <Th>
-                                                <BoldText>{localize('Currencies')}</BoldText>
-                                            </Th>
-                                            <Th>
-                                                {pd.is_crypto ? (
-                                                    <BoldText>{localize('Min deposit')}</BoldText>
-                                                ) : (
-                                                    <>
-                                                        <BoldText>{localize('Min - max')}</BoldText>
-                                                        <BoldText>{localize('deposit')}</BoldText>
-                                                    </>
-                                                )}
-                                            </Th>
-                                            <Th>
-                                                {pd.is_crypto ? (
-                                                    <BoldText>
-                                                        {localize('Min withdrawal')}
-                                                    </BoldText>
-                                                ) : (
-                                                    <>
-                                                        <BoldText>{localize('Min - max')}</BoldText>
+                        {payment_data.map((pd, idx) => {
+                            // eslint-disable-next-line no-console
+                            console.log({ ...payment_data }, pd, idx)
+
+                            return (
+                                <AccordionItem
+                                    key={idx}
+                                    style={{
+                                        padding: '2.2rem 4.8rem',
+                                        position: 'relative',
+                                        paddingBottom: pd.note ? '5rem' : '2.2rem',
```

Comment:
padding_bottom or paddingButtom?

Reply:
this is css in JS, it only accept name like that 🙏 , can take a look at this https://cssinjs.org/?v=v10.0.4 for reference